### PR TITLE
Unittests

### PR DESCRIPTION
**Pull request by philomath213**
see #3

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        out = lsb.cycle_lsb(192)
+
+        msg = ""
+        for i in range(0, len(out), 8):
+            msg += chr(int(out[i:i+8], 2))
+
+        self.assertEqual(msg, "LSB cycle hidden message")
+
+
+class TestZxing(unittest.TestCase):
```
Comment (youben11) [inline tests/tests.py:34]: does the tests here works for you?

Comment (philomath213) [reply tests/tests.py:34]: the test should work on both python versions 2 and 3, but the "Zxing" module use "HTMLParser" module which is available on python2 only.
we should re-implement the "Zxing" module and use "lxml" module to parse the html.

Comment (youben11) [reply tests/tests.py:34]: on it

File: tests/tests.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        result = zxing.decode(img_path)
+        raw_text = result["Raw text"]
+        self.assertEqual(raw_text, "http://en.m.wikipedia.org")
+
+
+class TestLcg(unittest.TestCase):
+
+    def test_lcg(self):
+        lcg = Lcg(0x66e158441b6995, 0xB, 1 << 85, 53)
+        state = lcg.get_state(2752470789, 3367609997, 1185935283)
```
Comment (youben11) [inline tests/tests.py:48]: this one is also failing, need an update to the LCG module

**Review by youben11 — CHANGES_REQUESTED** [review]

File: tests/tests.py (hunk 2 above)
Comment (philomath213) [reply tests/tests.py:48]: now it's ready to be merged

**Review by youben11 — APPROVED** [review]

**Pull request merged**